src/mono/mono/metadata/qcalllist.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _MONO_METADATA_QCALLLIST_H_
+#define _MONO_METADATA_QCALLLIST_H_
+
+extern const void* gPalGlobalizationNative[];
+extern const void* gEventPipeInternalFuncs[];
+
+
+typedef struct MonoQCallDef
+{
+    const char* class_name;
+    const char* namespace_name;
+    const void**  functions;
+} MonoQCallDef;
+
+typedef struct MonoQCallFunc {
+    int*            flags;
+    void*           implementation;
+    const char*     method_name;
+} MonoQCallFunc;
+
+enum {
+    func_flag_end_of_array   = 0x01,
+    func_flag_has_signature = 0x02,
+    func_flag_unreferenced = 0x04, // Suppress unused fcall check
+    func_flag_qcall        = 0x08, // QCall - mscorlib.dll to mscorwks.dll transition implemented as PInvoke
+};
```

Comment:
this doesn't really follow Mono's coding conventions (no `mono` prefix, etc).  Also we only need `0x01` and `0x08`, right?  could we leave out the other two: comment them out in case we need to find them later.

These flags are an implementation detail of the qcall table, right? We're not obligated to follow the coreclr implementation verbatim.  so we could make `MonoQCallFunc:flags` anything we want.

Reply:
Oh I see https://github.com/dotnet/runtime/blob/master/src/coreclr/src/libraries-native/entrypoints.c is shared with CoreCLR.  So we are tied to their naming and layout.

Let's have a comment about it